### PR TITLE
feat: add enum for class values

### LINKED ISSUE #15: Add an enum for all possible `_class` values
Likely required for https://github.com/sketch-hq/sketch-assistant-utils/issues/55 so that we have a data source to create all possible iterators.

### PR DESCRIPTION
**Pull request by jedrichards**
Resolves #15

### CODE REVIEW

**Review by HugoDaniel — APPROVED** [review]
Noice!

Cool dance with the `Set` guaranteeing uniqueness and the Boolean('') providing a typesafe filter on a typesafe map.

👍

File: tsconfig.json [hunk 1]
```diff
@@ -6,7 +6,9 @@
     "noUnusedParameters": true,
     "strict": true,
     "esModuleInterop": true,
-    "outDir": "dist"
+    "outDir": "dist",
+    "target": "ES6",
```
Comment (mathieudutour) [inline tsconfig.json:10]: that's a breaking change, please revert.

index.js goes from
```js
"use strict";
var __importStar = (this && this.__importStar) || function (mod) {
    if (mod && mod.__esModule) return mod;
    var result = {};
    if (mod != null) for (var k in mod) if (Object.hasOwnProperty.call(mod, k)) result[k] = mod[k];
    result["default"] = mod;
    return result;
};
exports.__esModule = true;
var FileFormat1 = __importStar(require("./v1-types"));
exports.FileFormat1 = FileFormat1;
var FileFormat2 = __importStar(require("./v2-types"));
exports.FileFormat2 = FileFormat2;
var FileFormat3 = __importStar(require("./v3-types"));
exports.FileFormat3 = FileFormat3;
exports["default"] = FileFormat3;
```
to 
```js
import * as FileFormat1 from './v1-types';
import * as FileFormat2 from './v2-types';
import * as FileFormat3 from './v3-types';
export { FileFormat1, FileFormat2, FileFormat3 };
export default FileFormat3;
```
for no reason and breaks all node tools that depends on it

Comment (jedrichards) [reply tsconfig.json:10]: 😱Thanks for the report. Will take a look!

Comment (jedrichards) [reply tsconfig.json:10]: Should be fixed in `4.0.5`.

**Pull request merged**